src/components/pages/CommunityPage/CommunityPage.tsx

```diff
@@ -24,14 +24,12 @@ export default function CommunityPage(): JSX.Element {
     <Button title={isShowMoreCmtrs ? 'Show less' : 'Show more'} variant="transparentWhite" onClick={() => setIsShowMoreCmtrs(!isShowMoreCmtrs)}/>
   );
 
-  const forbiddenUsers = ['jianghaiting', 'technoboy', 'linlin']
-  const currectUsers = ['Jason918', 'Technoboy-', '315157973']
-
-  const correctUsersExceptions = (element) => {
-    if(forbiddenUsers.indexOf(element.apacheId) !== -1){
-      element.apacheId = currectUsers[forbiddenUsers.indexOf(element.apacheId)];
-    }
-    return element
+  type apacheId = string
```

Comment:
```suggestion
  type ApacheId = string
```

The common TypeScript convention is to use capitalized CamelCase for type names.